src/comparator/enum_comparator.py

```diff
@@ -30,8 +30,8 @@ def compare(self):
         # 4. If the EnumDescriptors have the same name, check the values of them stay the same.
         # Enum values are identified by number, not by name.
```

Comment:
Just curious, why identify enum values by number instead of name?

Reply:
Enum value has a name and number, and they are both unique. And in the design [doc](https://docs.google.com/document/d/1vwUyzYGnC2FhA5s2y97W_oilskHp-wje-_R5LgJ2Mlo/edit) by @chrisdunelm, when comparing two enums, we decide to take number as identifier to pick enum values to compare, so that we can detect the name change.

Reply:
Aha, that makes sense. Thanks for explaining.